src/ServiceControl.Monitoring.UnitTests/Infrastructure/EndpointInstanceActivityTrackerTests.cs

```diff
@@ -17,7 +17,7 @@ public void Setup()
         [Test]
         public void When_no_endpoint_instance_reports_not_stale()
```

Comment:
`When_no_endpoint_instance_reports_not_stale` to `When_no_endpoint_instance_reports_stale`